tests/services/aws_sns_sqs_service_with_credentials.py

```diff
@@ -109,7 +124,7 @@ async def publish(data: Any, topic: str, **kwargs: Any) -> None:
 
         async def _async() -> None:
             async def sleep_and_kill() -> None:
-                await asyncio.sleep(30.0)
+                await asyncio.sleep(90.0)
```

Comment:
I had to raise the time limit for this test, since we are exercising retries here and the timeout is hardcoded (as far as I could see) to 30s.